x/pocketcore/keeper/aat.go

```diff
@@ -8,23 +8,28 @@ import (
 	pc "github.com/pokt-network/pocket-core/x/pocketcore/types"
 )
 
-// "AATGeneration" - Generates an application authentication token with an application public key hex string
-// a client public key hex string, a passphrase and a keybase. The contract is that the keybase contains the app pub key
-// and the passphrase corresponds to the app public key keypair.
-func AATGeneration(appPubKey string, clientPubKey string, key crypto.PrivateKey) (pc.AAT, sdk.Error) {
-	// create the aat object
+// GenerateAAT generates an AAT to be used for relay servicing.
+// - appPubKey is the public key of the application that's paying for on-chain service.
```

Comment:
Again, "paying" is an offchain concept.  We shouldn't say it in code.  Mainnet is not the only one network in the world.  From developer's perspective, appPubKey is just the public key of a staked app.

Reply:
Burn is disabled in Morse, but it is technically implemented (just old, untested, etc...).